packages/cozy-stack-client/src/FileCollection.js

```diff
@@ -661,6 +672,7 @@ class FileCollection extends DocumentCollection {
    * same doctype they have a few in common) here :
    * https://docs.cozy.io/en/cozy-doctypes/docs/io.cozy.files/#iocozyfiles
    *
+   * @private You shoud use udapte() directly.
```

Comment:
```suggestion
   * @private You shoud use update() directly.
```

Nit: I think we can move the docstring of updateAttributes to update.